tools/IoTEdgeDevOps/DevOpsLib/BugManagement.cs

```diff
@@ -11,12 +12,17 @@ namespace DevOpsLib
     public class BugManagement
     {
         const string WorkItemPathSegmentFormat = "{0}/{1}/{2}/_apis/wit/workitems/$Bug";
+        const string BackupOnCallReportLink = "https://microsoft.sharepoint.com/teams/Azure_IoT/_layouts/15/Doc.aspx?sourcedoc={1d485f94-0812-41c5-81b1-7fc0ca2dc9e4}&action=edit&wd=target%28Azure%20IoT%20Edge%2FServicing.one%7C0E88E43B-8A10-4A3F-982E-CF8B7441EAE8%2FBackup%20On-Call%20Reports%7Cfda0177f-3b35-4751-bfc9-7420cd56652c%2F%29&wdorigin=703";
```

Comment:
Do you have a document on how to obtain the link (in case of an update)?

Reply:
If the link breaks we can just replace it here. From the context in the bug it will be the `Backup On-Call Report` which everyone will be familiar with.